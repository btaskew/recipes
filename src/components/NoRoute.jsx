import React from 'react';

import Alert from 'react-bootstrap/lib/Alert';
import Page from 'components/Page';

function Heading() {
    return (
        <Page heading="No route found!">
            <Alert bsStyle="danger">404 - No route found!</Alert>
        </Page>
    );
}

export default Heading;
