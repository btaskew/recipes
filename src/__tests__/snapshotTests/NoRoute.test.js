import React from 'react';
import renderer from 'react-test-renderer';

import NoRoute from 'components/NoRoute';
import {MemoryRouter} from 'react-router';

describe('No route page', () => {
    it('renders correctly', () => {
        const noRoute = renderer
            .create(
                <MemoryRouter>
                    <NoRoute  />
                </MemoryRouter>
            )
            .toJSON();
        expect(noRoute).toMatchSnapshot();
    });
});
