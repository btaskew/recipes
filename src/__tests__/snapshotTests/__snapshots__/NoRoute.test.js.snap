// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`No route page renders correctly 1`] = `
Array [
  <div
    className="heading modal-header"
  >
    <h4
      className="modal-title"
    >
      No route found!
      <span
        className="float-right active glyphicon glyphicon-home"
        href="/"
        onClick={[Function]}
        style={Object {}}
      />
    </h4>
  </div>,
  <div
    className="modal-body"
  >
    <div
      className="alert alert-danger"
      role="alert"
    >
      404 - No route found!
    </div>
  </div>,
]
`;
